pkg/ccl/crosscluster/logical/logical_replication_job_test.go

```diff
@@ -1101,6 +1235,50 @@ func CreateScatteredTable(t *testing.T, db *sqlutils.SQLRunner, numNodes int, db
 	}, timeout)
 }
 
+func GetPGURLs(
+	t *testing.T, s serverutils.ApplicationLayerInterface, numDBs int,
+) ([]url.URL, func()) {
+	result := []url.URL{}
+	cleanups := []func(){}
+	for i := range numDBs {
+		dbName := string(rune('a' + i))
+		resultURL, cleanup := s.PGUrl(t, serverutils.DBName(dbName))
+		result = append(result, resultURL)
+		cleanups = append(cleanups, cleanup)
+	}
+
+	return result, func() {
+		for _, f := range cleanups {
+			f()
+		}
+	}
+}
+
+func VerifyExpectedRowAllServers(
+	t *testing.T, runners []*sqlutils.SQLRunner, expectedRows [][]string, numDBs int,
+) {
+	for i := range numDBs {
+		dbName := string(rune('a' + i))
+		runners[i].CheckQueryResults(t, fmt.Sprintf("SELECT * from %s.tab", dbName), expectedRows)
+	}
+}
+
+func WaitUntilReplicatedTimeAllServers(
+	t *testing.T,
+	targetTime hlc.Timestamp,
+	runners []*sqlutils.SQLRunner,
+	jobIDs [][]jobspb.JobID,
+	srcDB int,
+	numDBs int,
+) {
+	for targetDB := range numDBs {
```

Comment:
nit: instead of using `targetDB`, use `destDB`? I'd like to remove the term `target` from the terms we use because it is ambiguous.

Reply:
Done